.vscode/extensions.json

```diff
@@ -0,0 +1,69 @@
+		"dbaeumer.vscode-eslint",		 		  	//ESLint
+		"wayou.vscode-todo-highlight",			    //Todo Highlight	
+
+		// To contribute with Golang
+		"golang.go",                         		// Go language support
+
+		//To contribute with python
+		"ms-python.python",	                 		//Python
+		"ms-python.black-formatter",         		//To format the code with black
+		"slightc.pip-manager",               		//To install pip-free packages
+		"ms-python.isort",							//To sort the imports
+		"the-compiler.python-tox",					//To run the tests
+		"ms-python.pylint",							//To run the linter
+
+		//To contribute with docker
+		"ms-azuretools.vscode-docker",       		//Docker support
+		"ms-vscode-remote.remote-containers",		//To run the containers remotely with docker-machine and docker-compose 
+
+		//To contribute with HTML
+		"formulahendry.auto-close-tag",      		//To automatically close the tag when you type a tag name.
+		"formulahendry.auto-rename-tag",     		//To automatically rename the tag when you type a tag name.
+		"cstrap.flask-snippets", 		     		//To add snippets for flask framework in HTML editor.
+		"ecmel.vscode-html-css",			 		//To add snippets for html and css 
+		"zignd.html-css-class-completion",			//To add snippets for html and css classes completion
+		"solnurkarim.html-to-css-autocompletion",	//To add snippets for html to css autocompletion 
+
+		//To contribute with JS/TS
+		"xabikos.JavaScriptSnippets",		  		//To add snippets for JS/TS
+		"ms-vscode.vscode-typescript-next",	  		//To add snippets for TS next 
+
+		//To contribute with CSS/SCSS
+		"ritwickdey.LiveServer",			  		//To start a live server 
+		"glenn2223.live-sass",				  		//To format the code with sass 
+		"pranaygp.vscode-css-peek",			  		//To peek the CSS code 
+		"mrezechi3l.css-responsive",		  		//To format the code with responsive CSS
+		"kamikillerto.vscode-colorize",		  		//To colorize the code with syntax highlighting 
+
+		//To contribute with Angular
+		"johnpapa.Angular2",				  		//To add snippets for Angular 2 and Angular 2 routing (Angular 2 routing is not supported in vscode yet)
+		"Angular.ng-template",				  		//To add snippets for Angular 2 templates (ng-template) and ng-content (content) elements 
+		"alexiv.vscode-angular2-files",				//To add snippets for Angular files (component, directive, service, etc.) and templates (html, css, etc.) 
+		"natewallace.angular2-inline",				//To add snippets for Angular 2 inline templates (<ng-template>) and ng-content (<content>) elements
+		"humao.rest-client",						//To add snippets for RESTful API calls (GET, POST, PUT, DELETE, etc.) 
+
+		//To contribute with Terraform
+		"hashicorp.terraform",			  			//To add snippets for Terraform (provider, resource, etc.) 
+		"4ops.terraform",				  			//To add snippets for Terraform (provider, resource, etc.) 
+
+		//Newline at the end of json 
+		"ms-vscode.json-newline-at-end-of-file",	//To add newline at the end of json
```

Comment:
Not found

Reply:
I found the right name for the newline at the end of a file. I copy-pasted wrongly, it is now fixed.